### PR TITLE
refactor: remove nginx

### PR DESCRIPTION
**Pull request by shotonoff**
## Issue being fixed or feature implemented

Since there are 2 http-proxies (nginx and envoy) there was decided to simplify the approach of using one of them to route  gRPC, Web-gRPC and JSON RPC traffic to dash services. It allows us to maintain only one http-proxy.

## What was done?

Migrated nginx configuration to envoy

## How Has This Been Tested?

For testing used dashevo/platform-test-suite

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: configs/schema/configJsonSchema.js [hunk 1]
```diff
@@ -243,19 +234,23 @@ module.exports = {
...
                     },
-                    rate: {
+                    tokensPerFill: {
                       type: 'integer',
                       minimum: 0,
                     },
-                    burst: {
+                    fillInterval: {
+                      type: 'string',
+                    },
```
Comment (shumkov) [inline configs/schema/configJsonSchema.js:247]: Let's add `pattern: '^[0-9]+(.[0-9]+)?(m|s|h)$',`

File: configs/system/base.js [hunk 2]
```diff
@@ -46,21 +46,17 @@ module.exports = {
...
-          image: 'nginx:latest',
-        },
         rateLimiter: {
-          enable: true,
-          rate: 120,
-          burst: 300,
+          maxTokens: 300,
+          tokensPerFill: 150,
+          fillInterval: '60s',
+          enabledForPercentage: 100,
```
Comment (shumkov) [inline configs/system/base.js:59]: `enabled: true`

File: configs/schema/configJsonSchema.js (hunk 1 above)
Comment (shotonoff) [reply configs/schema/configJsonSchema.js:248]: Add also `ms` resolution

**Review by shotonoff — COMMENTED** [review]
did some changes according to feedback

File: configs/schema/configJsonSchema.js (hunk 1 above)
Comment (shumkov) [reply configs/schema/configJsonSchema.js:248]: I guess the question mark is unnecessary there.

**Review by shumkov — APPROVED** [review]
Good job! 👍

**Pull request merged**